docs/Installation.rst

```diff
@@ -114,147 +116,271 @@ This is only required if the unit tests are to be build.
+#. Download a version of LAPACK from `official website
+   <http://www.netlib.org/lapack/>`_ and extract the archive in some directory of your
+   filesystem.
 
-#. Configure compilation with cmake by typing::
+#. Create a build directory (e.g. Build) and configure the compilation with CMake by 
+   typing::
 
      mkdir Build
      cd Build
      cmake ./..
 
-#. If you don't have a root access on your machine or you want to install LAPACK
-   to  a custom directory, use ccmake to change CMAKE_INSTALL_PREFIX. In Build
+#. If you don't have root access on your machine or want to install LAPACK
+   to a custom directory, use ccmake to change CMAKE_INSTALL_PREFIX. In the Build
    directory type::
 
      ccmake ./..
      <Navigate to CMAKE_INSTALL_PREFIX and change it to your folder>
      Press [c] to configure
      Press [g] to generate and exit
 
-#. Now compile LAPACK and install by typing::
+#. Now compile LAPACK and install it by typing::
 
      make
      make install
 
+macOS
+-----
+
+Note: the installation tutorial for macOS assumes that you have root access to your
+machine and makes use of the `Homebrew` package manager; however, you may use a
+different package manager (e.g. `Anaconda`) if you are more familiar with it.
+
+#. Check that your Mac is running on macOS >= 15.0. You may check the version of your
+   operating system and update it if necessary by going into *System Settings* > *General* 
+   > *Software Update*.
+
+#. Install `Xcode` from the App Store. `Xcode` contains crucial headers which are read 
+   and interpreted when compiling software containing C/C++ languages. Once installed, 
+   launch `Xcode` so that it can complete its initialisation. A dialog will be presented 
+   indicating which Simulator runtimes are built-in, and which Simulator runtimes you may 
+   download. Choose `Continue` to finish setting up `Xcode`.
+
+#. Open a new `Terminal` window. If `Terminal` is not docked, you may find it by opening 
+   a new `Finder` window, then going to *Applications* > *Utilities*.
+
+#. Install `Homebrew` by typing the following command in your `Terminal` window::
+
+	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+#. Once `Homebrew` is installed, type the following command in your `Terminal` window. 
+   This will install the latest versions of all the packages required to correctly set 
+   up and run SCONE::
+
+     brew install gcc cmake python git openblas lapack libomp
+
+#. Close your `Terminal` window. Open a new `Finder` window and navigate to your `Home` directory 
+   (⌘ + ⇧ + h). Display hidden files (⌘ + ⇧ + .) and find the ``.zprofile`` file (this is the macOS 
+   equivalent of the ``.bashrc`` file on Linux distributions). Open it and insert **any of the 
+   following lines which are not already present** (note: this depends on whether you have a Mac 
+   running on an Intel CPU or an ARM - Apple Silicon - chip):
+   
+   * Intel::
```

Comment:
I don't quite follow why there are two sections based on the CPU ISA?
The only difference seem to be the install path of the `homebrew` package manager. Maybe we can combine both sections (and just point out the difference at the start) to minimise repetition?